SoomlaAndroidProfile/src/com/soomla/profile/social/ISocialProvider.java

```diff
@@ -19,12 +19,13 @@
 import android.app.Activity;
 
 import com.soomla.profile.auth.IAuthProvider;
+import com.soomla.profile.domain.IProvider;
 
 /**
  * A provider that exposes social capabilities such as sharing, fetching user
  * feeds, uploading images etc.
  */
-public interface ISocialProvider extends IAuthProvider {
+public interface ISocialProvider extends IProvider {
```

Comment:
@EugeneButusov I did confirm removing. My question is why does it need to extend anything at all?